menu/textmenu/PhyMenuInit.c

```diff
@@ -0,0 +1,32 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "include/boot.h"
+#include "TextMenu.h"
+#include "PhyMenuActions.h"
+
+TEXTMENU *PhyMenuInit(void) {
+	TEXTMENUITEM *itemPtr;
+	TEXTMENU *menuPtr;
+
+	menuPtr = malloc(sizeof(TEXTMENU));
+	memset(menuPtr,0x00,sizeof(TEXTMENU));
+	strcpy(menuPtr->szCaption, "Peripherals Menu");
+
+
+	itemPtr = malloc(sizeof(TEXTMENUITEM));
+	memset(itemPtr,0x00,sizeof(TEXTMENUITEM));
+	strcpy(itemPtr->szCaption, "Serial COM1: ");
```

Comment:
Is COM1 a suitable name?

Reply:
According to SMSC LPC47M157 spec it allows to have up to two serial ports, COM1 and COM2 accordingly.